android/app/src/main/java/com/reactnativenavigation/views/LightBox.java

```diff
@@ -54,6 +54,15 @@ private void createContent(final Context context, LightBoxParams params) {
         lightBox.setBackgroundColor(params.backgroundColor.getColor());
         lightBox.addView(content, lp);
 
+        if (params.tapBackgroundToDismiss) {
+            lightBox.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    hide();
```

Comment:
Both BottomTabsLayout/SingleScreenLayout both use the `.hide()` method.

If I use the `.destroy()` the dialog will close without an animation